web-app/js/portal/search/FacetedSearchResultsDataView.js

```diff
@@ -333,7 +326,10 @@ Portal.search.FacetedSearchResultsDataView = Ext.extend(Ext.DataView, {
 
             Portal.data.ActiveGeoNetworkRecordStore.instance().add(record);
         }
-        Ext.MsgBus.publish(PORTAL_EVENTS.VIEW_GEONETWORK_RECORD, record);
+
+        if (!multiSelect) {
+            Ext.MsgBus.publish(PORTAL_EVENTS.VIEW_GEONETWORK_RECORD, record);
```

Comment:
I think you're right about that event being not-quite-right. You'll also notice it's the only even that isn't saying *this thing happened* rather it's saying *do this thing*. I'll have a look at whether we can adjust the scope of that.

Yeah, I reckon we can do that rename. We still refer to geonetwork records (e.g. ActiveGeonetworkRecordStore) but we can start moving to using the data collection terminology.